neural_compressor/adaptor/pytorch.py

```diff
@@ -2671,6 +2683,10 @@ def quantize(self, tune_cfg, model, dataloader, q_func=None):
 
         # check smoothquant folding value
         recipe_cfgs = tune_cfg.get("recipe_cfgs", None)
+        if "smooth_quant_args" in recipe_cfgs and "auto_alpha_args" in recipe_cfgs["smooth_quant_args"]:
+            do_bias_shift = recipe_cfgs["smooth_quant_args"]["auto_alpha_args"].get("shift_bias", False)
```

Comment:
do->enable or choose alternative words, or consider removing the 'do' , 'do_blockwise' is similar